packages/recorder-loader/src/loader.ts

```diff
@@ -45,7 +53,7 @@ const unsafeLoadAndStartRecorder = ({
 
     // Setup configuration
     const typedWindow = window;
-    typedWindow.METICULOUS_RECORDING_TOKEN = projectId;
+    typedWindow.METICULOUS_RECORDING_TOKEN = recordingToken ?? projectId;
```

Comment:
Typings make `recordingToken` required so I guess we can just use `recordingToken` here?

Reply:
Oh that's true I guess

Reply:
Actually, since some clients are using JS not TS I'll change this back to how it was before